widget/embedded/src/components/ConfirmWalletsModal/ConfirmWallets.helpers.ts

```diff
@@ -1,9 +1,15 @@
-import type { BestRouteResponse, BlockchainMeta } from 'rango-sdk';
+import type {
+  BlockchainMeta,
+  MultiRouteSimulationResult,
+  SimulationResult,
+} from 'rango-sdk';
 
-export function getRequiredWallets(quote: BestRouteResponse | null): string[] {
+export function getRequiredWallets(
+  quote: MultiRouteSimulationResult | SimulationResult | null
+): string[] {
```

Comment:
I believe it's more effective to identify the shared characteristic among these types and only provide the function with the necessary data :
getRequiredWallets( swaps: SwapResult [ ] | null ) : string []

Reply:
fixed